server/routes/creatingLicences/handlers/confirmCreate.ts

```diff
@@ -50,7 +50,7 @@ export default class ConfirmCreateRoutes {
     const { answer } = req.body
     const backLink = req.session.returnToCase
 
-    if (answer === YesOrNo.YES) {
+    if (answer === YesOrNo.YES && !!nomisId) {
```

Comment:
can this be null? wouldn't the path fail to render if this was missing?

Reply:
Yes, if it is null then it will redirect to caseload page.
This is POST method is triggered from front end button click, so it should have the nomisId.